composer.json

```diff
@@ -1,27 +1,26 @@
 {
-    "name": "mettle/sendportal-core",
-    "homepage": "https://github.com/mettle/sendportal-core",
+    "name": "sabas/sendportal-core",
+    "homepage": "https://github.com/sabas/sendportal-core",
     "description": "Sendportal core functionality.",
     "type": "library",
     "license": "MIT",
     "authors": [],
     "require": {
-        "php": "^7.3 || ^8.0",
         "ext-json": "*",
-        "aws/aws-sdk-php-laravel": "^3.6",
-        "doctrine/dbal": "^3.0",
-        "illuminate/support": "^8.0",
+        "aws/aws-sdk-php-laravel": "^3.8",
+        "doctrine/dbal": "4.0.0-beta2",
```

Comment:
Is there a reason that you're targetting a beta version here?

Reply:
I looked at packagist and found the latest released version :) Actually in L10 it's not needed for migrations perhaps it can be removed?